application/models.py

```diff
@@ -106,10 +106,10 @@ class Meta:
 
     # Methods
     def get_absolute_url(self):
-        """Returns the url to access a particular instance of MyModelName."""
+        """Returns the url to access a particular instance of Patient."""
         return reverse('model-detail-view', args=[str(self.id)])
     
     def __str__(self):
-        """String for representing the MyModelName object (in Admin site etc.)."""
-        return self.name
+        """String for representing the Patient object (in Admin site etc.)."""
+        return self.last_Name
```

Comment:
almost

Reply:
What did I miss here? Sorry I might not be seeing it right since I'm on my phone too

Reply:
I think it should just be last_name. I can update it